system/cle/cle.c

```diff
@@ -798,6 +799,9 @@ static int cle_editloop(FAR struct cle_s *priv)
 
   for (; ; )
     {
+#if  1 /* Perhaps here should be a config switch */
```

Comment:
But @patacongo why we add #if/#endif?

Reply:
It was added.  It was already there.  Just copied.  The original authored intended to add a configuration option but left #if1 in place of the missing option.